### DISCUSSION TITLE
SF 12 issues

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#343] [General]
Has anyone had success with SF12 in Reticulum? I've had success with RNode control and basic packet transfer, but there are issues in Reticulum. 

SF12 BW 250k can't find a path and thus don't communicate, and SF12 BW 500k has a 60% announce loss (but good packet reception, strangely). The bit-rate appears sufficient, which confuses me. Has anyone had success or have any thoughts on why it appears to need a far higher data rate to function in Reticulum?

**Comment by markqvist** [comment]
Hmm, that is odd. It should work, and I see no reason that there should be anything wrong with the driver for that mode, but I will look into it more when I get a chance.

Any further input/experiences from anyone is very welcome.
